databoard/aws_backend.py

```diff
@@ -7,10 +7,9 @@
 from subprocess import call
 from subprocess import check_output
 
-from databoard.model import Submission
-from databoard.model import db
+from databoard import db
+from databoard.db_tools import Submission
```

Comment:
to be clear, this file was not being used anymore right? (then the changes are not that important to review)